### DISCUSSION TITLE
Only need a refetch to indexedDB, use mutate or revalidate?

### DISCUSSION THREAD

**Discussion by hanszhoou** [vercel/swr#686] [Q&A] [answered]
I am using SWR with indexedDB. 

Here's the scenario, very simple: I update one list item in indexedDB, after that I need to refresh the List by making an indexedDB query to get the list again. As the indexedDB request is very fast, I don't need to update the local data immediately. I just need a refetch. 

I found calling `mutate()` without parameters and calling `revalidate()` both work out, so I want to know which one is more suitable?

**Comment by sergiodxa** [comment, marked as answer]
Revalidate is deprecated, it’s preferable to only use mutate without parameters.

**Reply by Jack-Works** [reply]
Does it mean using `mutate` with no parameters do the same thing as what `revalidate()` do before?

**Reply by sergiodxa** [reply]
Yep, it will literally call revalidate and it will return the new revalidate data as a promise too
